build_tools/github_actions/run_benchmarks.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Not sure where this file should be put. It's a completely CI-only script so I put it under `build_tools/github_actions`

Reply:
Eh I don't think it's any more "CI only" than most of our other CI scripts. I'd put it in build_tools/benchmarks probably? There's nothing about it that's actually specific to github_actions (unlike the configure_ci.py script). We should probably reorganize the CI build scripts. They're basically all in the cmake/ directory, which I don't think actually helps much.

Reply:
Moved to `build_tools/benchmarks`. This script was using hard-coded `./build_tools/github_actions/docker_run.sh`. Refactored it so we can also run it locally.